hnn_core/viz.py

```diff
@@ -168,19 +168,34 @@ def plot_spikes_raster(cell_response, ax=None, show=True):
     import matplotlib.pyplot as plt
     spike_times = np.array(sum(cell_response._spike_times, []))
     spike_types = np.array(sum(cell_response._spike_types, []))
+    spike_gids = np.array(sum(cell_response._spike_gids, []))
     cell_types = ['L5_pyramidal', 'L5_basket', 'L2_pyramidal', 'L2_basket']
-    spike_times_cell = [spike_times[spike_types == cell_type]
-                        for cell_type in cell_types]
+    cell_type_colors = {'L5_pyramidal': 'r', 'L5_basket': 'b',
+                        'L2_pyramidal': 'g', 'L2_basket': 'w'}
 
     if ax is None:
         fig, ax = plt.subplots(1, 1)
 
-    ax.eventplot(spike_times_cell, colors=['r', 'b', 'g', 'w'])
-    ax.legend(cell_types, ncol=2)
+    count = 0
```

Comment:
could we call this guy `ypos` ? I was a bit confused with `count`

and update `whats_new.rst`. Then I'll merge. Thanks a ton!